compiler/codegen/FrontEnd.hpp

```diff
@@ -125,6 +125,7 @@ class TR_FrontEnd : public TR_Uncopyable
    // Method
    // --------------------------------------------------------------------------
 
+   virtual bool handleExceptionThrownFromILGen();   // This API is to be used in a catch block
```

Comment:
Should we do the rethrow in the catch block and return an enum saying yes, no, or rethrow? Thoughts?